inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/node/Application.java

```diff
@@ -390,10 +392,12 @@ public static void main(String[] args) {
                     application.handleConfigurationEvent(configurationProvider.getConfiguration());
                 }
             }
-            //metrics
+            // metrics
             MetricObserver.init(ConfigManager.getInstance().getCommonProperties());
-            
-            //start application
+            // audit
+            AuditUtils.initAudit();
+
```

Comment:
When the program exits, it need to call AuditImp.getInstance().sendReport() to send the audit data of the memory

Reply:
add sendReport in hook